lib/subscriptions_test_kit/suites/subscriptions_r5_backport_r4_client/common/subscription_simulation_utils.rb

```diff
@@ -138,6 +157,10 @@ def set_subscription_reference(subscription_status, subscription_url)
       def find_parameter(subscription_status, parameter_name)
         subscription_status.parameter&.find { |p| p.name == parameter_name }
       end
+
+      def find_parameter_list(subscription_status, parameter_name)
```

Comment:
Nit picking, but this method name seemed a little misleading to me, I'd probably go with `filter_parameters` or `find_all_parameters` instead, mirroring the name of the ruby method (for which `select` is an alias)

Reply:
fixed!